space-plugins/nuxt-base/layouts/default.vue

```diff
@@ -1,10 +1,15 @@
 <script setup lang="ts">
 const config = useAppConfig();
-const { status } = useAppBridge();
+const { completed, appBridgeAuth, oauth } = useAppBridge();
+
+const nuxtApp = useNuxtApp();
+nuxtApp.provide('appBridge', {
+	completed,
+	appBridgeAuth,
+	oauth,
+});
 </script>
 
 <template>
-	<slot v-if="!config.appBridge.enabled || status === 'authenticated'" />
-	<div v-else-if="status === 'authenticating'"></div>
-	<div v-else-if="status === 'error'"></div>
+	<slot v-if="!config.appBridge.enabled || completed" />
```

Comment:
I've actually done slightly more than just refactoring.

I'm providing `appBridge` here, so that users can access `$appBridge` in their projects.

Reply:
So this basically create app bridge globally accessible in the frontend? @eunjae-lee